src/nrnpython/nrnpy_hoc.cpp

```diff
@@ -2884,6 +2884,64 @@ static void sectionlist_helper_(void* sl, Object* args) {
   }
 }
 
+/** value of neuron.coreneuron.enable as 0, 1 (-1 if error) */
```

Comment:
as discussed over the call, I don't have better suggested to simplify this. Could we just add comment with `TODO` saying this should be letter improved? (just as reminder)

Reply:
Added:
```
 *  TODO: seems like this could be generalized so that
 *  additional cases would require less code.
```